app/vue/package.json

```diff
@@ -9,16 +9,16 @@
   "license": "MIT",
   "main": "dist/client/index.js",
   "bin": {
-    "build-storybook": "./dist/server/build.js",
-    "start-storybook": "./dist/server/index.js",
-    "storybook-server": "./dist/server/index.js"
+    "build-storybook": "./bin/build.js",
+    "start-storybook": "bin/index.js",
+    "storybook-server": "bin/index.js"
```

Comment:
This change is needed, because yarn needs some files to link as binaries, see https://github.com/storybooks/storybook/pull/1810/files#r137413061